tesseract-server/src/app.rs

```diff
@@ -18,16 +18,24 @@ use crate::handlers::{
 use std::sync::{Arc, RwLock};
 
 
+#[derive(Debug, Clone)]
+pub struct EnvVars {
+    pub flush_secret: Option<String>,
+    pub database_url: String,
+    pub schema_filepath: Option<String>,
```

Comment:
Can we make this not an `Option`? I think we need the schema filepath no matter what. Or, are you preparing for eventually wanting to support the dynamic schema refresh from api? If so, I'd probably make the schema filepath a value in an enum `SchemaSource` or something, with the other value `schema_refresh_endpoint` or something. Let me know what you think.